src/components/SearchResultDetail/SearchResultHtmlModal.test.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import { render, waitFor, screen } from '@testing-library/react';
+
+import SearchResultHtmlModal from './SearchResultHtmlModal';
+
+describe('renders the raw html of search result', () => {
```

Comment:
It should match the component name:

```suggestion
describe('SearchResultHtmlModal' () => {
```

or:

```suggestion
describe('Search Result Html Modal' () => {
```

In any case, all component tests need to follow the same naming pattern.